### DISCUSSION TITLE
Where can I find a place to detect visible bounding box, full bounding box?

### DISCUSSION THREAD

**Discussion by hoya-cho** [open-mmlab/mmyolo#816] [General]
Where can I find a place to detect visible bounding box, full bounding box?
While examining the crowdhuman dataset, I found that the 1st and 2nd stage detectors detect visible bounding boxes and full bounding boxes. But I'm using yolo8 in mmyolo. Where can I find the code to get the two bounding boxes above?

**Comment by josh3255** [comment]
Does full bounding box mean before nms?
